ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -7941,11 +7942,57 @@ def test_torch_instance_topk(
         init_all_as_kwargs_np={"data": input[0]},
         method_input_dtypes=input_dtype,
         method_all_as_kwargs_np={
-            "k": k, "dim": axis, "largest": largest, "sorted": sorted},
+            "k": k,
+            "dim": axis,
+            "largest": largest,
+            "sorted": sorted,
+        },
```

Comment:
Can we revert this back?

Reply:
Hi @ZiadAmerr, so this was done automatically by the pre-commit. Possibly, the corresponding contributor missed the pre-commit. I kept it as we follow the pre-commit rules. We can revert it tho no issues. Let me know what you think.

Reply:
No we can leave it, since it shouldn't cause any errors and it is a linting improvement anyways